lib/Digitick/Sepa/TransferInformation/CustomerDirectDebitTransferInformation.php

```diff
@@ -44,7 +44,7 @@ class CustomerDirectDebitTransferInformation extends BaseTransferInformation
     /**
      * @var bool
      */
-    protected $amendedDebtorAccount = false;
+    protected $amendedDebtorAgent = false;
```

Comment:
As stated above this breaks BC I follow you in that it makes sense but do you really want to introduce a BC break for this change ? I recommend to create a separate PR for this and we can make a newer version which will break BC and clean up some other stuff together with this